lib/amazonka/src/Amazonka/Auth/ConfigFile.hs

```diff
@@ -224,6 +238,9 @@ data ConfigProfile
   | -- | Recognizes @role_arn@, @role_session_name@, and
     -- @web_identity_token_file@.
     AssumeRoleWithWebIdentity Text (Maybe Text) FilePath
+  | -- | Recognized @sso_start_url@, @sso_region@, @sso_account_id@, and
+    -- @sso_role_name@.
+    SSO Text Region Text Text
```

Comment:
Maybe we call this `AssumeRoleViaSSO` for consistency with the other constructors?

Reply:
Yeah I think it makes sense conceptually, even though it's not an `sts:AssumeRole` or `sts:AssumeRoleWithWebIdentity` call.